pkg/steps/template.go

```diff
@@ -90,6 +107,10 @@ func (s *templateExecutionStep) Run(ctx context.Context, dry bool) error {
 		}
 	}
 
+	if err := operateOnTemplatePods(s.template, s.artifactDir); err != nil {
+		log.Printf("errors while operating in template pods: %v", err)
```

Comment:
@stevekuznetsov Should we consider the error a reason not to continue?

Reply:
It will make sense not to continue because the artifact container is not injected in the pod.